eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -1,16 +1,30 @@
 parameters: 
   TargetVariable: ''
+  TargetUserVariable: 'notspecified'
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
   ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
 
 steps:
-  - pwsh: |
-      git clone https://github.com/Azure/azure-sdk-tools.git $(Build.SourcesDirectory)/tools_repo
-      cd $(Build.SourcesDirectory)/tools_repo
-      git checkout azure-sdk-tools_20210114.1
-    displayName: Setup Identity Resolver
+  - task: DotNetCoreCLI@2
+    displayName: 'Install Identity Resolver'
+    inputs:
+      command: custom
+      custom: 'tool'
+      arguments: 'install --global --add-source "${{ parameters.DevOpsFeed }}" --version "1.0.0-dev.20211018.1" "Azure.Sdk.Tools.IdentityResolver"'
```

Comment:
Why are these versions different? We should pull this version into a variable and share it across both of these. 

We also can consider combining these tools into one tools package.